vizro-core/tests/unit/vizro/models/test_layout.py

```diff
@@ -146,12 +144,3 @@ def test_working_grid(self, grid):
             vm.Layout(grid=grid)
         except ValidationError as ve:
             assert False, f"{grid} raised a value error {ve}."
-
-
-@pytest.mark.parametrize("grid", [[[0, -1], [1, 2]], [[0, -1, 1, 2]], [[-1, -1, -1], [0, 1, 2]]])
```

Comment:
Didn't change the test, I've just moved this one to `test_models_utils` as the helper function is in the `_models_utils.py` file